_posts/2016-07-26-No-Time-To-Wait-Preservation-FFV1-Matroska-Symposium.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+title:  "No Time to Wait: Standardizing FFV1 and Matroska for Preservation"
+date:   2016-07-26 08:00:00 CET
+categories: mediaconch, presentations, amia
+---
+
+# No Time to Wait: Standardizing FFV1 and Matroska for Preservation
+
+![first day](/MediaConch/images/nttw1.png)
```

Comment:
Yeah it's because it's preparing to live in a place with /MediaConch as the subdomain rather than being top-level. Renders if running locally and --baseurl='/MediaConch'. It's ... a workaround.